### DISCUSSION TITLE
I'm not clear - clips vs recordings?

### DISCUSSION THREAD

**Discussion by ksignorini** [blakeblackshear/frigate#2364] [Ask A Question]
Maybe I'm just tired, but what's the difference between clips and recordings?

**Comment by blakeblackshear** [comment]
The best explanation is probably the 0.9.1 release notes https://github.com/blakeblackshear/frigate/releases/tag/v0.9.1

Essentially, there is no such thing as "clips" in current versions. Frigate records video from your camera. You can choose to keep all of the video or only the parts that overlap with events.

**Comment by ksignorini** [comment]
Hmm. OK, I just read that. 

So why are there separate clips and recordings folders? What goes in each?

**Reply by blakeblackshear** [reply]
The clips folder is a legacy name. It only contains snapshots in current versions. It will eventually be renamed to snapshots.
